force_wfmanager/ui/setup/setup_pane.py

```diff
@@ -299,6 +299,9 @@ def sync_selected_mv(self):
         self.selected_mv = self.task.side_pane.workflow_tree.selected_mv
         if self.selected_mv is not None:
             if isinstance(self.selected_mv.model, BaseModel):
+                #: FIXME - this will raise an AttributeError if
+                #: self.selected_mv.model has not initialised a traits_view
+                #: object
                 self.selected_model = self.selected_mv.model
```

Comment:
If `DataSourceModelView` listens to all traits on a `DataSouceModel`, this returns:

`AttributeError: 'ProbeDataSourceModel' object has no attribute 'traits_view'`

For certain selected models.

Reply:
This comment is now outdated